content/shmem_sync_all.tex

```diff
@@ -0,0 +1,44 @@
+\color{ForestGreen}
+\apisummary{
+    Registers the arrival of a \ac{PE} at a barrier and suspends \ac{PE}
```

Comment:
This sounds familiar, did we discuss this recently in the context of a different proposal that updated the barrier text?  IIRC, we preferred "suspends execution" over "blocks".